src/components/Modals/RenameProjectModal.js

```diff
@@ -47,39 +40,20 @@ export const RenameProjectModal = () => {
     renameProjectMutation({variables: {id: project.id, name: newName}, onCompleted: onCompleted})
   }
```

Comment:
🔧 worth a quick dig but this action fails for me locally, I get an Apollo error 🤔 and I can't check it in staging because login fails. Can you quickly check this is ok your end?

error:
```
Uncaught (in promise) ApolloError: Identifier has already been taken
```

Reply:
Yeah not sure what's happening with this as it's working at my end 🤔

Reply:
yea this was me... not sure what/why/how but a fresh pull and rebuild worked 👍